### ISSUE TITLE
Any more than 2 strategies in strategies.js is an issue

### ISSUE THREAD

**Issue by xteejx** [DustinJSilk/reactive-trader#6]
I'm not quite sure why this happens, but with more than 2 strategies in the strategies.js file and the GA just doesn't work.
There might also be a requirement on nested parameters within custom strategies that also cause 500 internal server errors.
Can't quite put my finger on which it is, or both but seems to be working again now just fine.

**Comment by DustinJSilk** [comment]
Just tried running DEMA, RSI, and MACD. Ran as expected.

Could you give it a go with the latest code and if it still gives errors, post the contents of the latest error log found in `reactive-trader/logs/xxx-error`.

Theres some more info on the logging files at the bottom  of  [this](https://github.com/DustinJSilk/reactive-trader/blob/master/ISSUE_TEMPLATE.md) file if you need

**Comment by xteejx** [comment]
Okay, seems to be working this end. It seems candleWriter must explicitly be set to enabled. Makes sense as it writes to the database.....
Still a bit dodgy with latest pull of exchange info on Kraken though, and needs starting a couple of times, but working nonetheless.

**Comment by xteejx** [comment]
Right, after quite a bit of variation testing, it looks like it's a problem purely with kraken importing. Problem with 'date' is undefined and no trades available. Likely to be a gekko core issue. Doesn't help as it's the only place I trade :(

**Comment by DustinJSilk** [comment]
Ah okay thats interesting. Ive been testing with Poloniex only, figured Gekko would handle them all the same.

Ill give Kraken a look this week. If its a problem with Gekko we'll need to submit the issue on their side.

I'll also check out the deal with candleWriter and force set it where needed.

**Comment by xteejx** [comment]
Cool beans. Thanks mate :)
